package.json

```diff
@@ -50,5 +50,8 @@
     "bracketSpacing": true,
     "jsxBracketSameLine": true,
     "arrowParens": "always"
+  },
+  "dependencies": {
+    "recharts": "^2.4.3"
```

Comment:
This should a dependency for webcert only, revert this change and add with `yarn workspace @frontend/webcert add recharts`